README.md

```diff
@@ -8,10 +8,7 @@ In the European Union, this extension's fields correspond to the following [eFor
 
 * BT-660 (Framework Re-estimated Value)
 * BT-709 (Framework Maximum Value)
-* BT-150 (Contract identifier)
 * BT-708 (Documents Official Language)
-* BT-67 (Exclusion Grounds)
-* BT-22 (Internal Identifier)
 
 For correspondences to eForms fields, see OCDS for eForms.
```

Comment:
```suggestion
For correspondences to eForms fields, see [OCDS for eForms](https://standard.open-contracting.org/profiles/eforms/latest/en/).
```
I know the link is already in the first section but for completeness it feels like it should be repeated here